brainstormforce/ultimate-addons-for-gutenberg
# Improvement: Added multiple new settings in multi button block

### Description
Added multiple new settings in multi-button block.

Parent Button Block Improvement -

- We should add all stack option ✅
- Button alignment ( Responsive option ) ✅
- Typography Component ✅

Child Button block Improvement -

- Remove text Option ✅
- Box Shadow Option ✅
- Background Option ✅
- Typography ✅
- Margin Option ✅
- Icon color ✅
- Button Size ( Small | Medium | Large ) ✅

### Test Cases - 

- Switch to master branch
- Drag and drop multi-buttons block
- try different layouts and styling options
- Switch to new-btn-block branch
- Check editor for block validation issue
- update the editor and check front-end

Note - We intentionally broke Front-end Button UI for backword.

### Test case -

- Switch to new-btn-block
- Drag and drop multi buttons block
- check styling and settings for different devices
- Check on front-end whether it is working properly or not

### Types of changes
breaking change which adds functionality

### How has this been tested?


### Checklist:
